mlir/lib/Conversion/GPUCommon/OpToFuncCallLowering.h

```diff
@@ -31,9 +31,9 @@ template <typename SourceOp>
 struct OpToFuncCallLowering : public ConvertOpToLLVMPattern<SourceOp> {
 public:
   explicit OpToFuncCallLowering(LLVMTypeConverter &lowering, StringRef f32Func,
-                                StringRef f64Func)
+                                StringRef f64Func, StringRef f32FastFunc)
       : ConvertOpToLLVMPattern<SourceOp>(lowering), f32Func(f32Func),
-        f64Func(f64Func) {}
+        f64Func(f64Func), f32FastFunc(f32FastFunc) {}
```

Comment:
Can you update the class documentation please?

Reply:
Sure, updated, the diff please see : [link](https://github.com/llvm/llvm-project/pull/99890/files#diff-f9d186a175b0bff9446d83449269fefcf90dd8f9171227afa2ee4fbbd7ce2ef6R18)